tray/src/com/dvt/main/creational/builder/Client.java

```diff
@@ -0,0 +1,16 @@
+<<<<<<< HEAD
+package com.dvt.creational.builder;
+=======
+package creational.builder;
+>>>>>>> main
```

Comment:
Please fix merge conflict.
```suggestion
<<<<<<< HEAD
package com.dvt.creational.builder;
=======
package creational.builder;
>>>>>>> main
```